src/Microsoft.DocAsCode.Common/Path/RelativePath.cs

```diff
@@ -312,6 +312,48 @@ public bool Equals(RelativePath other)
             string.Concat(Enumerable.Repeat(ParentDirectory, _parentDirectoryCount)) +
             string.Join("/", _parts);
 
+        /// <summary>
+        /// Test whether a relative path starts with another folder relative path
+        /// </summary>
+        public bool InDirectory(RelativePath value)
+        {
+            if (value == null)
+            {
+                throw new ArgumentNullException(nameof(value));
+            }
+            if (value._parts[value._parts.Length - 1] != "")
+            {
+                return false;
+            }
+            if (_isFromWorkingFolder ^ value._isFromWorkingFolder)
+            {
+                return false;
+            }
+            if (_parentDirectoryCount > 0 || value._parentDirectoryCount > 0)
```

Comment:
>_parentDirectoryCount [](start = 16, length = 21)

is `../a/b` in directory `../a/`?

Reply:
It's hard to determine whether `../` is in `../../`, so I return false id either path starts with `../`